libs/components/layout/src/lib/modules/toolbar/toolbar.component.ts

```diff
@@ -19,4 +27,15 @@ export class SkyToolbarComponent {
   ) {
     this.hasSections = !!value && value.length > 0;
   }
+
+  /**
+   * A descriptor term for the items that the toolbar actions are manipulating. This term should be plural and is used to construct context specific accessibility labels for the search inputs, sort menus, and filter buttons.
```

Comment:
I played around with rearranging and tweaking the text for this entry and with aligning it with similar descriptions elsewhere. This is what I'd suggest for the full description here:

A descriptor for the items that the toolbar manipulates. Use a plural term. The descriptor helps set the toolbar’s `aria-label` attributes for search inputs, sort menus, and filter buttons to provide text equivalents for screen readers [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility). For example, when the descriptor is “constituents,” the search input’s `aria-label` is “Search constituents.” For more information about the `aria-label` attribute, see the [WAI-ARIA definition](https://www.w3.org/TR/wai-aria/#aria-label).

Reply:
That look good to me. I have updated to your suggestion.